app/src/main/java/com/globant/scriptsapadea/ui/activities/MainActivity.java

```diff
@@ -19,6 +23,23 @@
 @ContentView(R.layout.activity_main)
 public class MainActivity extends BaseActivity {
 
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        //DBTesting
+        super.onCreate(savedInstanceState);
```

Comment:
Why you are testing in here ? If you want to create a test do it inside androidTest folder.